### ISSUE TITLE
[ERROR] OctoPi broken DL button?

### ISSUE THREAD

**Issue by bolsoncerrado** [guysoft/OctoPi#537]
Tried to download OctoPi 0.15 today, got this instead:

<Error>
<Code>AccessDenied</Code>
<Message>Access denied.</Message>
<Details>
Anonymous caller does not have storage.objects.get access to octoprint/raspberry/stable/octopi-stretch-lite-0.15.1.zip.
</Details>
</Error>

**Comment by guysoft** [comment]
Can yoi please link ti the download button you are using?  It works for me on my phone just now.

**Comment by bolsoncerrado** [comment]
The one on the OctoPi page. Let me try again.

**Comment by bolsoncerrado** [comment]
Just tried again, same error. Tries to access:

https://storage.googleapis.com/octoprint/raspberry/stable/octopi-stretch-lite-0.15.1.zip

**Comment by bolsoncerrado** [comment]
Perhaps as the owner you're able to access it? The error mentions "Anonymous caller":

Anonymous caller does not have storage.objects.get access to octoprint/raspberry/stable/octopi-stretch-lite-0.15.1.zip.

**Comment by bolsoncerrado** [comment]
This XML file does not appear to have any style information associated with it. The document tree is shown below.

**Comment by John-Mc** [comment]
I just tried it from the Download button on this page: https://github.com/guysoft/OctoPi
It worked fine for me.
I get the same error when using the link that @bolsoncerrado used.

**Comment by bolsoncerrado** [comment]
The error comes from this button on this page:

https://octoprint.org/download/

**Comment by bolsoncerrado** [comment]
...and I just retried and it works now :?

**Comment by John-Mc** [comment]
That link works for me, however, when I click on the link in your post, I get the same error you did.

**Comment by guysoft** [comment]
Thia works: https://storage.googleapis.com/octoprint/stable/octopi-stretch-lite-0.15.1.zip

Cc @foosel the /stable is the one displayed in the nightly pages. While raspberypi/stable is what on the download link

**Comment by foosel** [comment]
@guysoft weird. The prior one was at `raspberry/stable`. Any changes in your upload scripts?

In any case, should be fixed in the load balancing config:

```
$ curl -I https://octopi.octoprint.org/latest
HTTP/1.1 302 Moved Temporarily
Server: nginx
Date: Wed, 30 May 2018 15:03:15 GMT
Content-Type: text/html
Content-Length: 154
Connection: keep-alive
Location: https://storage.googleapis.com/octoprint/stable/octopi-stretch-lite-0.15.1.zip

$ curl -I https://storage.googleapis.com/octoprint/stable/octopi-stretch-lite-0.15.1.zip
HTTP/2 200
expires: Wed, 30 May 2018 16:03:30 GMT
date: Wed, 30 May 2018 15:03:30 GMT
cache-control: public, max-age=3600
last-modified: Sat, 26 May 2018 11:03:40 GMT
etag: "4600d24c0618376c2a10c1507b8ba990"
[...]
content-type: application/zip
content-disposition: attachment;filename="octopi-stretch-lite-0.15.1.zip";filename*=UTF-8''octopi-stretch-lite-0.15.1.zip
[...]
```
